packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactions/CoinmarketTransaction/CoinmarketTransactionInfo.tsx

```diff
@@ -31,13 +25,11 @@ export const CoinmarketTransactionInfo = ({ trade }: CoinmarketTransactionInfoPr
     const tradeType = translationString(translationKeys[trade.tradeType]).toUpperCase();
 
     return (
-        <CoinmarketTransactionText>
-            <CoinmarketTransactionMedium>
-                <Row flexWrap="wrap">
-                    {tradeType} • <FormattedDate value={date} date time /> •{' '}
-                    <CoinmarketTransactionStatus trade={trade} />
-                </Row>
-            </CoinmarketTransactionMedium>
-        </CoinmarketTransactionText>
+        <Text margin={{ top: spacings.xs }} variant="tertiary" typographyStyle="label">
+            <Row flexWrap="wrap">
```

Comment:
Does this need Row? Wouldn't it be the same if you do just <Text>{`${..} ${...} ...`}</Text>